packages/quantic/cypress/e2e/default-1/case-classification/case-classification-expectations.ts

```diff
@@ -164,13 +164,13 @@ function caseClassificationExpectations(selector: CaseClassificationSelector) {
         .logDetail('should log the "ticket_classification_click" UA event');
     },
 
-    fetchClassificationsAfterValueChange: () => {
+    fetchClassifications: () => {
       cy.wait(InterceptAliases.CaseClassification).logDetail(
         'should fetch new case classifications after the value changes'
```

Comment:
if you rename the expectation, do we want this detail to be updated as well then?

Reply:
sure!